PowerEditor/src/WinControls/ColourPicker/WordStyleDlg.cpp

```diff
@@ -747,6 +765,9 @@ void WordStyleDlg::setVisualFromStyleList()
 		_pBgColour->setColour(style._bgColor);
 		_pBgColour->setEnabled((style._colorStyle & COLORSTYLE_BACKGROUND) != 0);
 		isEnable = true;
+		int eolfilled = 0;
+		eolfilled = style._eolFilled;
+		::SendMessage(_hCheckEolFilled, BM_SETCHECK, eolfilled, 0);
 	}

```

Comment:
Why not skip the intermediate variable with a single statement like `::SendMessage(_hCheckEolFilled, BM_SETCHECK, (style._eolFilled ? 1 : 0), 0);`?